cmd/admin-server-info.go

```diff
@@ -25,6 +25,8 @@ import (
 	"github.com/minio/minio/pkg/disk"
 	"github.com/minio/minio/pkg/madmin"
 	"github.com/minio/minio/pkg/mem"
+
+	cpuinfo "github.com/shirou/gopsutil/cpu"
```

Comment:
Any specific reason to change the import name for the package. AFAICS it's better to be cpu.Info than cpuinfo.Info

Reply:
Ther is a conflict in import as `"github.com/minio/minio/pkg/cpu"` is also used.  Renamed the alias to `cpuhw`